internal/workflow_testsuite.go

```diff
@@ -66,6 +66,7 @@ type (
 	TestWorkflowEnvironment struct {
 		workflowMock mock.Mock
 		activityMock mock.Mock
+		nexusMock    mock.Mock
```

Comment:
It doesn't look like you are calling `AssertExpectations` on this mock like we do for `workflowMock` or `activityMock` is that correct? I think you also didn't add a `AssertNexusOperationCalled` or update the other `Assert*Called`